ref: document `dvc artifacts get` (#4809)
iterative/dvc.org: content/docs/command-reference/artifacts/index.md

# artifacts

Commands for working with DVC

> [artifacts](/doc/user-guide/project-structure/dvcyaml-files#artifacts) and the

Reviewer: ```suggestion
<abbr>artifacts</abbr> and the
```

> DVC [model registry](/doc/use-cases/model-registry).

Reviewer: ```suggestion
<abbr>model registry</abbr>.
```

## Synopsis

```usage
usage: dvc artifacts [-h] [-q | -v] {get} ...

positional arguments:
  COMMAND
    get          Download an artifact from a DVC project.
```

## Description

> `dvc artifacts` subcommands provide a command line client for working with model
> registry artifacts.
> 
> <admon icon="book">
> 
> See [Model Registry](/doc/use-cases/model-registry) for more info.
> 
> </admon>

Reviewer: ```suggestion
`dvc artifacts` subcommands provide a command line client for working with <abbr>model
registry</abbr> artifacts.
```
